### PR TITLE
Renamed custom themes to custom skins

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

File: docs/articles/custom-skins.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+2. Change some of the color values around.
+
+3. Change the selected skin from the appearance settings page.
+
+Files will now use your custom skin. If you add more skins, you can switch between them using the dropdown.
+
+<details>
+<summary>
+Template
```
Comment (chingucoding) [inline docs/articles/custom-skins.md:19]: How about "default template"?

File: docs/articles/custom-skins.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+1. Open your text editor of choice, and paste in the template located below. Then, save the file as ```%userprofile%\AppData\Local\Packages\49306atecsolution.FilesUWP_et10x9a9vyk8t\LocalState\Skins\test1.xaml```.
+
+2. Change some of the color values around.
+
+3. Change the selected skin from the appearance settings page.
+
+Files will now use your custom skin. If you add more skins, you can switch between them using the dropdown.
+
+<details>
```
Comment (chingucoding) [inline docs/articles/custom-skins.md:17]: Have you checked that the web version of this page also works correctly?

Comment (yaichenbaum) [reply docs/articles/custom-skins.md:17]: I tested locally, I can't test the template section though until it's deployed.

Comment (chingucoding) [reply docs/articles/custom-skins.md:17]: It also works on the website, tested locally.

Out of curiosity, why can't you test until it's deployed?

Comment (yaichenbaum) [reply docs/articles/custom-skins.md:17]: I don't have the repo cloned locally right now 🙃.

Comment (chingucoding) [reply docs/articles/custom-skins.md:17]: Haha fair enough.

**Pull request merged**